src/Lachain.Core/RPC/HTTP/Web3/TransactionServiceWeb3.cs

```diff
@@ -189,6 +189,16 @@ private string SendRawTransaction(string rawTx)
             }
         }
 
+        [JsonRpcMethod("fe_sendRawTransactionBatch")] 
```

Comment:
I think it is better to rename it to `la_sendRawTransactionBatch`, `fe` was for `FrontEnd`,  we don't use ths function in frontend app
```suggestion
        [JsonRpcMethod("la_sendRawTransactionBatch")] 
```